lms/djangoapps/course_wiki/views.py

```diff
@@ -38,7 +38,9 @@ def course_wiki_redirect(request, course_id, wiki_path=""):
     as it's home page. A course's wiki must be an article on the root (for
     example, "/6.002x") to keep things simple.
     """
-    course = get_course_by_id(CourseKey.from_string(course_id))
+    course_key = CourseKey.from_string(course_id)
+    course = get_course_by_id(course_key)
+    check_if_course_org_in_site_org_filters(course_key)
```

Comment:
Why are we course objects when check_if_course_org_in_site_org_filters is also doing this?

Reply:
changed the function to use `is_course_org_same_as_site_org` from edly app, now the course object is only fetched in the view.